### PR TITLE
fixing several pack validation errors

### PR DESCRIPTION
**Pull request by roysagi**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/23712 

## Description
fixing the following packs:
 ‘Expanse’,
 ‘ExtraHop’,
 ‘CommonReports’, - this requires next sdk release to pass 
 ‘Gmail’,
 ‘GmailSingleUser’,
 ‘JoeSecurity’,
 ‘Legacy’,
Lost_Stolen_Device
Indeni

### CODE REVIEW

File: Packs/CommonReports/Reports/report-last7DaysIncidentReport.json [hunk 1]
```diff
@@ -838,7 +838,7 @@
   },
   "endingDate": "0001-01-01T00:00:00Z",
   "sensitive": false,
-  "recipients": null,
+  "recipients": [],
```
Comment (ronykoz) [inline Packs/CommonReports/Reports/report-last7DaysIncidentReport.json:841]: what's the impact of this

File: Packs/CommonReports/Reports/report-ShiftSummary.json [hunk 2]
```diff
@@ -116,6 +116,7 @@
     "runOnce": false,
     "scheduled": false,
     "paperSize": "A4",
```
Comment (ronykoz) [inline Packs/CommonReports/Reports/report-ShiftSummary.json:118]: what's the impact of this

File: Packs/JoeSecurity/Playbooks/playbook-Detonate_File_From_URL_-_JoeSecurity.yml [hunk 3]
```diff
@@ -362,3 +362,5 @@ outputs:
 - contextPath: DBotScore.Score
   description: The actual score (only in case of report type=json).
   type: number
+tests:
```
Comment (ronykoz) [inline Packs/JoeSecurity/Playbooks/playbook-Detonate_File_From_URL_-_JoeSecurity.yml:365]: pretty sure we have a test for this

File: Packs/CommonReports/Reports/report-ShiftSummary.json (hunk 2 above)
Comment (roysagi) [reply Packs/CommonReports/Reports/report-ShiftSummary.json:118]: this is unchanged

File: Packs/CommonReports/Reports/report-last7DaysIncidentReport.json (hunk 1 above)
Comment (roysagi) [reply Packs/CommonReports/Reports/report-last7DaysIncidentReport.json:841]: that is a good question, the schema in the sdk required an array here and I could not find a way to make it accept null. will try and figure if this is a problem.

File: Packs/JoeSecurity/Playbooks/playbook-Detonate_File_From_URL_-_JoeSecurity.yml (hunk 3 above)
Comment (roysagi) [reply Packs/JoeSecurity/Playbooks/playbook-Detonate_File_From_URL_-_JoeSecurity.yml:365]: I see that `detonatefile_-_joesecurity` has a test = `Detonate File - Generic Test`

should this playbook use the same test?

File: Packs/CommonReports/Reports/report-last7DaysIncidentReport.json (hunk 1 above)
Comment (ronykoz) [reply Packs/CommonReports/Reports/report-last7DaysIncidentReport.json:841]: let's just upload them and see if we get different results

File: Packs/JoeSecurity/Playbooks/playbook-Detonate_File_From_URL_-_JoeSecurity.yml (hunk 3 above)
Comment (ronykoz) [reply Packs/JoeSecurity/Playbooks/playbook-Detonate_File_From_URL_-_JoeSecurity.yml:365]: i think that your filter is not catching it or something, need to make sure we are in the clear here

File: Packs/CommonReports/Reports/report-last7DaysIncidentReport.json (hunk 1 above)
Comment (roysagi) [reply Packs/CommonReports/Reports/report-last7DaysIncidentReport.json:841]: checked - worked the same as the existing one

**Pull request merged**